# OrderOrCancel.srv
bool order
bool cancel
int8 table_id
---
# Response part
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages
